src/components/summary/GroupsDisplay.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import GroupCard from './GroupCard'
+
+const GroupsDisplay = () => {
+  return (
+    <div className='flex flex-nowrap gap-4 overflow-x-auto pb-4 w-full
+    [&::-webkit-scrollbar]:h-[5px] [&::-webkit-scrollbar-thumb]:rounded-full [&::-webkit-scrollbar-thumb]:bg-gray-400'>
+      <GroupCard name="AntAlmanac" />
+      <GroupCard name="ZotMeet" />
+      <GroupCard name="Some group" />
+      <GroupCard name="Some group" />
+      <GroupCard name="Some group" />
+      <GroupCard name="Some group" />
+    </div>
+  )
+}
+
+export default GroupsDisplay
```

Comment:
```suggestion
export const GroupsDisplay = () => {
  return (
    <div className='flex flex-nowrap gap-4 overflow-x-auto pb-4 w-full
    [&::-webkit-scrollbar]:h-[5px] [&::-webkit-scrollbar-thumb]:rounded-full [&::-webkit-scrollbar-thumb]:bg-gray-400'>
      <GroupCard name="AntAlmanac" />
      <GroupCard name="ZotMeet" />
      <GroupCard name="Some group" />
      <GroupCard name="Some group" />
      <GroupCard name="Some group" />
      <GroupCard name="Some group" />
    </div>
  )
}
```
nit: prefer non-default exports